docs/user-guide/ze-usage.md

```diff
@@ -84,23 +23,18 @@ When working in remote or virtualized environments &mdash; such as Eclipse Che,
 
    Credentials will be stored on a per profile/per panel basis until one of the following takes place:
 
-   - Data Sets/USS/Jobs tree refresh caused by an update to the `zowe.config.json` file
+   - **Data Sets**/**USS**/**Jobs** tree refresh caused by an update to the `zowe.config.json` file
    - Zowe Explorer refresh in the **Command Palette**
    - Reload of the Visual Studio Code window
    - Closing and reopening the VS Code window
 
 ### Disabling Secure Credential Storage of credentials
-
-#### Zowe Explorer v2
+**[should there be a V3 section here?]**
```

Comment:
@adam-wolfe @t1m0thyj  please respond

Reply:
I think the instructions would be the same for V2 and V3.